doc/page-cmakefunctions.h

```diff
@@ -0,0 +1,123 @@
+/**
+
+\page CMake-Functions FaceLift CMake API Reference
+
+\section public-functions Functions for FaceLift Users
+
+The following functions are public and are meant to be used by FaceLift users.
+
+\code
+facelift_add_interface(<TargetName>
+                       INTERFACE_DEFINITION_FOLDER dir
+                       [LINK_LIBRARIES lib1 [lib2...]]
+                       [IMPORT_FOLDERS folder1 [folder2...]])
+\endcode
+
+This function adds a library containing the C++ classes generated from some QFace interface definition files
+
+\param TargetName Name of the library target to be added
+\param INTERFACE_DEFINITION_FOLDER The folder containing the QFace interface definition files from which the library should be generated
+\param LINK_LIBRARIES List of libraries that will be linked with this interface library
+\param IMPORT_FOLDERS List of folders with QFace files that will be imported
+
+<hr><br>
+
+\code
+facelift_generateQRC(<OutputFile>
+                     INPUT_FOLDERS folder1 [folder2...])
+\endcode
+
+Generates a qrc resource file from the given input folders. This function will look for js, qml, ttf, png, sci, json, astcz, tcsh, tesh, gsh, fsh files.
+
+\param OutputFile The qrc resource file that will be generated
+\param INPUT_FOLDERS List of folders containing resources to be added
+
+<hr><br>
+\code
+
+facelift_add_qml_plugin(<TargetName>
+                        URI uri
+                        [VERSION version]
+                        [OUTPUT_BASE_DIRECTORY dir]
+                        [SOURCES src1 [src2...]]
+                        [HEADERS hdr1 [hdr2...]]
+                        [SOURCES_GLOB [Globbing expressions]...]
+                        [HEADERS_GLOB [Globbing expressions]...]
+                        [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                        [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                        [HEADERS_NO_INSTALL hdrn1 [hdrn2...]]
+                        [HEADERS_GLOB_NO_INSTALL [Globbing expressions]...]
+                        [HEADERS_GLOB_RECURSE_NO_INSTALL [Globbing expressions]...]
+                        [LINK_LIBRARIES lib1 [lib2...]]
+                        [NO_INSTALL])
+\endcode
+
+This function adds a Qml plugin.
```

Comment:
Mention that the parameter descriptions can be found below ?

Reply:
done